libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/targets/js/webpack/KotlinWebpack.kt

```diff
@@ -131,8 +132,7 @@ constructor(
     internal var resolveFromModulesFirst: Boolean = false
 
     @get:OutputFile
-    open val configFile: Provider<File> =
-        npmProjectDir.map { it.resolve("webpack.config.js") }
+    abstract val configFile: RegularFileProperty
```

Comment:
Could we please leave `configFile` as is to not change type of the field?
Instead I propose to add new property `configurationFile` with type `RegularFileProperty` with convention value `npmProjectDir.map { it.resolve("webpack.config.js") }`.
It should look like something like that
```
    @get:OutputFile
    open val configurationFile: RegularFileProperty = objects.fileProperty()
        .convention(
            objects.fileProperty().fileProvider(
                npmProjectDir.map { it.resolve("webpack.config.js") }
            )
        )
```

And change `configFile` from `@get:OutputFile` to `@get:Deprecated` (because we are going to have new property as output file)